packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -59,6 +59,7 @@ import { TechDocsAddons } from '@backstage/plugin-techdocs-react';
 import { ReportIssue } from '@backstage/plugin-techdocs-module-addons-contrib';
 
 import { EntityTerraformContent } from '@globallogicuki/backstage-plugin-terraform';
+import { EntityTerraformLatestRunCard } from '@globallogicuki/backstage-plugin-terraform/src/plugin';
```

Comment:
This should come from the same import as line 61 above

Reply:
I get the following error message when I attempt to import it as recommended:

'"@globallogicuki/backstage-plugin-terraform"' has no exported member named 'EntityTerraformLatestRunCard'. Did you mean 'EntityTerraformCard'?ts(2724)

Reply:
You need to export it in `src/index.ts`

Reply:
Done